client-spark/shuffle-manager-3/src/main/scala/org/apache/spark/shuffle/rss/RssSparkUtils.scala

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Shall we merge this file with SparkUtils.java? I think the only difference between them is java and scala.

Reply:
Do you want to include those reflection utils into this project? it has been adopted by Apache Parquet, Iceberg and Kyuubi.
https://github.com/apache/incubator-kyuubi/tree/master/kyuubi-common/src/main/java/org/apache/kyuubi/reflection